storage-tool.yml

```diff
@@ -416,7 +434,62 @@ services:
       - name: max_file_size
         class: 4 TB
         notes:
+      - name: storage_warmth
+        class: hot
+        notes:
 
+  - name: Hibernate
+    description: |
+      [Hibernate is a secure, reliable, research data archive solution](https://docs.ccv.brown.edu/hibernate/). Hibernate is a Brown OIT archival service for the research community to migrate inactive data off active Network-attached storage (NAS) platforms onto a lower cost, long-term retention environment. 
```

Comment:
maybe make the link just on `Hibernate`?

Reply:
Yep! Fixed! Thank you, @mcmcgrath13!!